tools/NuGet/DynamoVisualProgramming.Core/Core.nuspec

```diff
@@ -2,19 +2,25 @@
 <package >
   <metadata>
     <id>DynamoVisualProgramming.Core</id>
-    <version>1.0.0-beta1</version>
+    <version>1.0.0-beta3</version>
     <authors>Autodesk</authors>
     <owners>Autodesk</owners>
     <licenseUrl>http://opensource.org/licenses/MIT</licenseUrl>
     <projectUrl>https://github.com/DynamoDS/Dynamo</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Unit and system test infrastructure for Dynamo.</description>
+    <description>Unit and system test infrastructure for Dynamo. It contains the following files:
+		1)DynamoCore.dll
+		2)ProtoCore.dll
+		3)DynamoShapeManager.dll 
+		4)DynamoUtilities.dll
+	</description>
     <summary>This package contains the core assemblies for Dynamo.</summary>
     <copyright>Copyright Autodesk 2016</copyright>
     <iconUrl>https://raw.githubusercontent.com/DynamoDS/Dynamo/master/doc/distrib/Images/logo_square_32x32.png</iconUrl>
     <dependencies>
         <dependency id="Prism" version="4.1"/>
         <dependency id="NUnit" version="2.6.3"/>
+		<dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="1.0.0-beta3"/>
```

Comment:
@sharadkjaiswal why do we need for Core to have a dependency on ZeroTouchLibrary?